### ISSUE TITLE
Examples broken

### ISSUE THREAD

**Issue by brianmay** [IdentityPython/pysaml2#449]
Hello,

There seem to be a number of problems getting the examples to work under Python 3.6.x.

For example:

```
Traceback (most recent call last):
  File "./idp.py", line 11, in <module>
    from cherrypy import wsgiserver
ImportError: cannot import name 'wsgiserver'
Traceback (most recent call last):
  File "./sp.py", line 852, in <module>
    from cherrypy import wsgiserver
ImportError: cannot import name 'wsgiserver'
```

Regards

**Comment by c00kiemon5ter** [comment]
Latest `cherrypy` release move the `wsgiserver` to another component (namely `cheroot`). 
The correct behaviour would be to just `import wsgiserver`.

**Comment by brianmay** [comment]
I just noticed this was already mentioned in #413.

**Comment by brianmay** [comment]
Also, #406 is a problem.

**Comment by c00kiemon5ter** [comment]
I am closing this, as it should be fixed by #477
